bigquery_etl/glam/templates/clients_daily_scalar_aggregates_v1.sql

```diff
@@ -0,0 +1,100 @@
+{{ header }}
+
+WITH extracted AS (
+  SELECT
+    *,
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    REPLACE(ping_info.ping_type, "_", "-") AS ping_type,
+    COALESCE(
+      SAFE_CAST(SPLIT(client_info.app_display_version, '.')[OFFSET(0)] AS INT64),
+      0
+    ) AS app_version,
+    client_info.os AS os,
+    client_info.app_build AS app_build_id,
+    client_info.app_channel AS channel
+  FROM
+    `moz-fx-data-shared-prod.{{ source_table }}`
+  WHERE
+    DATE(submission_timestamp) = {{ submission_date }}
+    AND client_info.app_channel IN ("release", "fenixProduction")
+    AND client_info.client_id IS NOT NULL
+),
+unlabeled_metrics AS (
+  SELECT
+    {{ attributes }},
+    ARRAY<STRUCT<metric STRING, metric_type STRING, key STRING, agg_type STRING, value FLOAT64>>[
+        {{ unlabeled_metrics }}
+    ] as scalar_aggregates
+  FROM
+    extracted
+  GROUP BY
+    {{ attributes }}
+),
+grouped_labeled_metrics AS (
```

Comment:
Note: If the existing separation between scalars and keyed-scalars were to be emulated, it could be done using`{% if is_labeled %} ... {% endif %} sections with a variable on whether to include certain sub-expressions.